modules/convert/notification.go

```diff
@@ -54,16 +58,20 @@ func ToNotificationThread(n *models.Notification) *api.NotificationThread {
 			}
 		}
 	case models.NotificationSourceCommit:
+		url := n.Repository.HTMLURL() + "/commit/" + n.CommitID
 		result.Subject = &api.NotificationSubject{
-			Type:  api.NotifySubjectCommit,
-			Title: n.CommitID,
-			URL:   n.Repository.HTMLURL() + "/commit/" + n.CommitID,
+			Type:    api.NotifySubjectCommit,
+			Title:   n.CommitID,
+			URL:     url,
+			HTMLURL: url,
 		}
 	case models.NotificationSourceRepository:
 		result.Subject = &api.NotificationSubject{
 			Type:  api.NotifySubjectRepository,
 			Title: n.Repository.FullName(),
-			URL:   n.Repository.Link(),
+			// FIXME: this is a relative URL, rather useless and inconsistent, but keeping for backwards compat
```

Comment:
sorry :/ if v1.15.0 would not be out we could just fix it - now i think it is more les useles? make it consisten to return api url?